core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -36,6 +36,21 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
               }
             }
         } ~
+          pathPrefix("billing-account") {
+            pathEnd {
+              put {
+                entity(as[UpdateRawlsBillingAccountRequest]) { updateProjectRequest =>
+                  complete( userServiceConstructor(userInfo).UpdateBillingAccount(RawlsBillingProjectName(projectId), updateProjectRequest) )
+                }
+              } ~
+                delete {
+                  complete {
+                    userServiceConstructor(userInfo).DeleteBillingAccount(RawlsBillingProjectName(projectId))
+                  }
+                }
+            }
+
```

Comment:
should we have some unit tests for the routing, and/or the put and delete functionality?

Reply:
Absolutely, I will add those.